cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -376,6 +386,31 @@ def copy(self, default=None):
 
         return result
 
+    def action_show_host_key(self):
```

Comment:
I think it would be reasonable to split this into two separate functions: one get's the host key, another shows it.
In case we may need to get the host key later in some other situations.